homeassistant/components/tuya/translations/select.uk.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
This file doesn't belong in this PR. Translations can be provided, after this PR is merged, via the Lokalise translation platform.

See: <https://developers.home-assistant.io/docs/translations>

Please remove this file.

Reply:
Ok, I'll do it.

Reply:
Done